<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addRoomDialog</class>
 <widget class="QDialog" name="addRoomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nest</string>
  </property>
  <widget class="QComboBox" name="comboBox_type">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string> 单人房</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string> 双人房</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string> 家庭房</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string> 豪华套房</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>房间类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>价格：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_price">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;Calibri&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_roomnumber">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;Calibri&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>房间号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:hover
{
font: 12pt &quot;华文楷体&quot;;
color:rgb(117, 117, 117);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:pressed
{
font: 12pt &quot;华文楷体&quot;;
color:rgb(182, 182, 182)
border:2px groove gray;border-radius:10px;padding:2px 4px
}
</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:hover
{
font: 12pt &quot;华文楷体&quot;;
color:rgb(117, 117, 117);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:pressed
{
font: 12pt &quot;华文楷体&quot;;
color:rgb(182, 182, 182)
border:2px groove gray;border-radius:10px;padding:2px 4px
}
</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
